package.json

```diff
@@ -33,21 +33,23 @@
     "url": "https://github.com/ssense/vue-carousel/issues"
   },
   "homepage": "https://github.com/ssense/vue-carousel#readme",
+  "dependencies": {
+    "@babel/polyfill": "^7.0.0-beta.53",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
You should remove this one, b/c it's redundant with `@babel/core` and shouldn't be in `dependencies`, since it's only for development purposes.